doc/authors.rst

```diff
@@ -61,3 +61,4 @@
 .. _William Turner: https://bootstrapbill.github.io/
 .. _Yorguin Mantilla: https://github.com/yjmantilla
 .. _Julius Welzel: https://github.com/JuliusWelzel
+.. _Kalle Mäkelä: https://github.com/Kallemakela
```

Comment:
This file is meant to be alphabetical by first name. Not sure why @JuliusWelzel occurs twice; please remove line 63 and move yourself to just before Kambiz.